packages/tasks/src/snippets/js.ts

```diff
@@ -78,9 +93,11 @@ for await (const chunk of stream) {
 	}  
 }`,
 				},
-				{
-					client: "openai",
-					content: `import { OpenAI } from "openai";
+				...(provider === "hf-inference"
+					? [
```

Comment:
hmm my brain is friend but i think we can still support this for external providers (it's the same API)

I can do it if you want @SBrandeis

Reply:
Ah yes, feel free, to give it a try!

Reply:
8aafe2562642db47cc2c8e38b6f2deadf711c81b